coalib/bearlib/aspects/taste.py

```diff
@@ -55,6 +55,14 @@ def __init__(self, description: str='', suggested_values: tuple=(),
         given `languages`, which must be language identifiers supported by
         :class:`coalib.bearlib.languages.Language`.
 
+        :param description:      str     A string describing taste.
+        :param suggested_values  tuple   A tuple containing the list of
+                                         possible values for the taste.
+        :param default           bool    The default value of taste.
+        :param languages         tuple   A tuple containing list of
+                                         languages, for which taste is
+                                         defined.
+
```

Comment:
Also I don't think the types for each parameter are required. They are already mentioned in the signature